### PR TITLE
Redirect README to wiki

### PR DESCRIPTION
**Pull request by rbillings**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1 @@
...
-
-`npm test`
-
-## Distributing
-
-To distribute, you will need AMO access credentials. See the `web-ext` docs.
-
-1. Use [`web-ext
-   sign`](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/web-ext_command_reference#web-ext_sign)
+[Link to permanent Blok QA test plan](https://wiki.mozilla.org/QA/Blok)
```
Comment (pdehaan) [inline README.md]: We may just want to add a `## QA Test Plan` section near the bottom.

Also, what's the difference between the wiki.moz test plan, and the doc we have at https://github.com/mozilla/blok/blob/master/docs/testplan.md?

Or did we mean to replace the docs/testplan.md doc with the wiki link (versus modifying the main README.md here)?

**Comment by groovecoder** [comment]
Yeah, we don't want to replace the entire `README.md` file with the link; we just want to replace `docs/testplan.md` file with the link.

**Comment by rbillings** [comment]
This test plan was the original and the wiki version will be the maintained official test plan.  Since we don't want to maintain two separate test plans this was intended.

**Comment by pdehaan** [comment]
LGTM! 👍

**Pull request merged**